Homework4.3/homework/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+
+class Program
+{
+    static void Main()
+    {
+        
+        int elelemntsArray = 28; 
+
+        // Создаём массив для четных чисел
+        int[] evenArray = new int[elelemntsArray];
+        // Создаём массив для нечетных чисел
+        int[] oddArray = new int[elelemntsArray];
+
+        // Заполняем массивы четными и нечетными числами
+         for (int i = 0; i < elelemntsArray; i++)
+         {
+             evenArray[i] = 2 * i;
+             oddArray[i] = 2 * i + 1;
```

Comment:
We need to fill main array with Random and then fill evenArray and oddArray with the filtered values. This logic is incorrect